imessage/bluebubbles/api.go

```diff
@@ -1227,12 +1227,12 @@ func (bb *blueBubbles) wsUrl() string {
 	q.Add("transport", "websocket")
 	u.RawQuery = q.Encode()
 
-	url := u.String()
+	URL := u.String()
```

Comment:
Local variables (and in general anything that isn't exported from the package) should start with lowercase, so this was correct already before

Reply:
I noticed I was changing the scope on some variables with this commit (and undid that change) https://github.com/mautrix/imessage/pull/189/commits/0df68f50ee1626f56ea373c112763225af1ea610 but missed this one. 

I'll refactor that so we're not unintentionally exporting anything.

Reply:
Variables inside functions are never exported, so having them start with uppercase is always wrong

Reply:
Oh I see, even if they're an initial. Thanks for clarifying that!

Reply:
[This](https://github.com/mautrix/imessage/pull/189#discussion_r1494294407) PR should address it.